SilKit/source/services/logging/Logger.cpp

```diff
@@ -80,10 +277,29 @@ class SilKitRemoteSink
 
 Logger::Logger(const std::string& participantName, Config::Logging config)
     : _config{std::move(config)}
-{
+{  
     // NB: do not create the _logger in the initializer list. If participantName is empty,
     //  this will cause a fairly unintuitive exception in spdlog.
-    _logger = spdlog::create<spdlog::sinks::null_sink_st>(participantName);
+    for (auto sink : _config.sinks)
+    {
+        if (sink.type == Config::Sink::Type::Remote)
+        {
+            _loggerRemote = std::make_shared<RemoteLogger>(sink.level, participantName); 
+        }
+        else
+        {
+            if (sink.format == Config::Sink::Format::Json 
+                && nullptr == _loggerJson)
+            {
+                _loggerJson = spdlog::create<spdlog::sinks::null_sink_st>(participantName + "_Json");
```

Comment:
... use only the participant name for the JSON and ...

```suggestion
                _loggerJson = spdlog::create<spdlog::sinks::null_sink_st>(participantName);
                spdlog::drop(participantName);
```